command/forceleave/forceleave_test.go

```diff
@@ -56,6 +56,42 @@ func TestForceLeaveCommand(t *testing.T) {
 	})
 }
 
+func TestForceLeaveCommand_prune(t *testing.T) {
+	t.Parallel()
+	a1 := agent.NewTestAgent(t, t.Name(), ``)
+	defer a1.Shutdown()
+	a2 := agent.NewTestAgent(t, t.Name(), ``)
+	defer a2.Shutdown()
+
+	_, err := a2.JoinLAN([]string{a1.Config.SerfBindAddrLAN.String()})
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	// Forcibly shutdown a2 so that it appears "failed" in a1
+	a2.Shutdown()
+
+	ui := cli.NewMockUi()
+	c := New(ui)
+	args := []string{
+		"-http-addr=" + a1.HTTPAddr(),
+		"-prune",
+		a2.Config.NodeName,
+	}
+
+	code := c.Run(args)
+	if code != 0 {
+		t.Fatalf("bad: %d. %#v", code, ui.ErrorWriter.String())
+	}
+	retry.Run(t, func(r *retry.R) {
+		m := len(a1.LANMembers())
+		if m != 1 {
+			r.Fatalf("should have 1 members, got %#v", m)
```

Comment:
you might also want to check that the remaining member is the correct member just for sanity